qiskit/providers/aer/backends/wrappers/CMakeLists.txt

```diff
@@ -26,11 +26,28 @@ if(APPLE)
     unset(PYTHON_LIBRARIES)
 endif()
 
+set(_find_pybind_includes_command "
+import sys
+import pybind11
+sys.stdout.write(pybind11.get_include())
+")
+execute_process(COMMAND "${PYTHON_EXECUTABLE}" -c "${_find_pybind_includes_command}"
+                OUTPUT_VARIABLE _py_output
+                RESULT_VARIABLE _py_result)
+if(_py_result EQUAL "0")
+    message(STATUS "PYCOMM RAW: ${_py_output}")
+    set(PYBIND_INCLUDE_DIRS "${_py_output}")
+    message(STATUS "PYBIND INCLUDES FOUND: ${PYBIND_INCLUDE_DIRS}")
+else()
+    message(FATAL_ERROR "COULD NOT FIND PYBIND!")
+endif()
 
 # QASM Controller
 
-add_cython_target(qasm_controller_wrapper qasm_controller_wrapper.pyx CXX)
-add_library(qasm_controller_wrapper MODULE ${qasm_controller_wrapper})
+add_library(qasm_controller_wrapper MODULE qasm_bindings.cc)
+
+#add_cython_target(qasm_controller_wrapper qasm_controller_wrapper.pyx CXX)
```

Comment:
if you can make it work as the doc says, we don't need to comment any code, just remove it. (We always have git history if we want to take it back)

Reply:
I removed the commented sections